pyproject.toml

```diff
@@ -68,6 +68,9 @@ pytest-xdist = ">= 2.4.0"
 mocket = { version = "^3.10.4", extras = ["speedups"] }
 Faker = "^11.3.0"
 pyfakefs = "^4.5.4"
+testcontainers = "^3.4.2"
```

Comment:
This is the wrong group. testcontainers is a production dependency of the project.